docs/src/main/tut/docs/learning_resources.md

```diff
@@ -10,15 +10,13 @@ Here are some various learning resources concerning optics. Feel free to send a
 ### Presentations
 -   [Beyond Lenses](https://www.youtube.com/watch?v=6nyGVgGEKdA): how `Iso`, `Prism`, `Lens` and `Optional`
 relate to each other (by [Julien Truffaut](https://github.com/julien-truffaut))
+-   [Optics with Monocle: Modeling The Part and The Whole](https://www.youtube.com/watch?v=NvCcNM2vp3k)
```

Comment:
I think you should be able to edit the PR from your browser, if not I will do it